inlang/source-code/sdk/src/loadProject.ts

```diff
@@ -309,6 +458,10 @@ export async function loadProject(args: {
 			errors: createSubscribable(() => [
 				...(initializeError ? [initializeError] : []),
 				...(resolvedModules() ? resolvedModules()!.errors : []),
+				...(loadMessagesViaPluginError() ? [loadMessagesViaPluginError()!] : []),
```

Comment:
Those are the errors that we should report in fink as well (for clarification with @NiklasBuchfink )

Reply:
per @NiklasBuchfink all errors are handled and shown in Fink
added #2358 to improve Fink validation of error handling